### PR TITLE
Update Markdown Files

### LINKED ISSUE #268: Documentation update Kumaran
1. SettingUp.md
2. AboutUs.md
3. Index.md

### PR DESCRIPTION
**Pull request by kumsssss**
1) Added repo link in SettingUp.md
2) Changed intro in index.md
3) Added acknowledgement of AB3 in index.md
4) Updated AboutUs file to have everyone's emails

Closes  #268

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -15,40 +14,49 @@ You can reach us at the email `e0406389@u.nus.edu`
...
-* Roles and Responsibilities: Logic expert, Code Quality, Documentation
+* Role: Developer
+* Responsibilities: Logic expert, Code Quality, Documentation
 
 ### Mohamed Fazil
 
 <img src="images/md-fazil.png" width="200px">
 
-[[github](http://github.com/Md-Fazil)] [[portfolio](team/mohamedfazil.md)]
+[[github](http://github.com/Md-Fazil)] [[portfolio](team/mohamedfazil.md)] You can reach me at the email `mohamedfazil@u.nus.edu`
```
Comment (Md-Fazil) [inline docs/AboutUs.md:37]: Perhaps, the line "You can reach me at the email `mohamedfazil@u.nus.edu`" should be in the following line.

**Review by Md-Fazil — COMMENTED** [review]
Minor formatting changes needed before merging.

File: docs/AboutUs.md (hunk 1 above)
Comment (kumsssss) [reply docs/AboutUs.md:37]: they will all appear in the same line so I don't think it matters

File: docs/AboutUs.md [hunk 2]
```diff
@@ -15,40 +14,49 @@ You can reach us at the email `e0406389@u.nus.edu`
 
 [[github](https://github.com/JayChenYJ)]
 [[portfolio](team/jaychenyj.md)]
+You can reach me at the email `e0406389@u.nus.edu`
```
Comment (JayChenYJ) [inline docs/AboutUs.md:17]: put a spacing between portfolio and this sentence, so will be in a new paragraph :)

**Review by JayChenYJ — APPROVED** [review]

**Pull request merged**